src/LitePatch/LitePatch.Ui/Components/Pages/PatchApply.razor

```diff
@@ -0,0 +1,49 @@
+﻿@page "/PatchApply"
+@using LitePatch.Services.Interfaces
+@using LitePatch.Ui.Components.Controls
+@using LitePatch.Services.Models
+@inject IGitPatchService GitPatchService
+
+<MudPaper Elevation="2">
+    <MudStack>
+    <MudText>Apply Patches to:</MudText>
+        <FolderPicker 
```

Comment:
The repo should use the repo from the settingsservice, but we can use this folderpicker to choose the folder where patches are read from instead.